submissions/Ant-C-tech/oop-exercise/index.js

```diff
@@ -0,0 +1,105 @@
+/* Refer to https://github.com/OleksiyRudenko/a-tiny-JS-world for the task details
+   Complete the below for code reviewers' convenience:
+
+   Code repository: https://github.com/Ant-C-tech/a-tiny-JS-world
+   Web app: _put project's github pages URL here_
+   */
+
+// ======== OBJECTS DEFINITIONS ========
+// Define your objects here
+
+class Inhabitant {
+    constructor(name, gender, friends) {
+        //At the moment I can't imagine what the creatures will inhabit our world, that is why I assumed that these will be some people and different kinds of their pets. In this way I can assume that they all would have these common properties and one method to perform the task.
```

Comment:
The OOP compliant code is easier to update when requirements change.
Stick to current requirements, i.e. make code good to meet them.